fix: display the number that was negative when adding an emoji (#372)
gsoft-inc/ov-igloo-ui: .changeset/clever-taxis-nail.md

---
'@igloo-ui/textarea': patch
---

> fix display the number that was negative when adding an emoji

Reviewer: ```suggestion
Fix the display of the negative character count when adding an emoji
```
